panel/src/main/scala/hmda/panel/PanelCsvLoader.scala

```diff
@@ -0,0 +1,101 @@
+package hmda.panel
+
+import java.io.File
+
+import akka.{ Done, NotUsed }
+import akka.actor.ActorSystem
+import akka.util.{ ByteString, Timeout }
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.{ FileIO, Flow, Framing, Sink, Source }
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.{ HttpEntity, _ }
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import com.typesafe.config.ConfigFactory
+import hmda.apiModel.protocol.admin.WriteInstitutionProtocol
+
+import scala.concurrent.duration._
+import hmda.parser.fi.InstitutionParser
+import org.slf4j.LoggerFactory
+import spray.json._
+
+import scala.util.Try
+
+object PanelCsvLoader extends WriteInstitutionProtocol {
+  implicit val system: ActorSystem = ActorSystem("hmda")
+  implicit val materializer = ActorMaterializer()
+  implicit val timeout: Timeout = Timeout(5.second)
+  implicit val ec = system.dispatcher
+  val log = LoggerFactory.getLogger("hmda")
+
+  val config = ConfigFactory.load()
+  val host = config.getString("hmda.adminHost")
+  val port = config.getInt("hmda.adminPort")
+
+  def main(args: Array[String]): Unit = {
+
+    if (args.length < 1) {
+      exitSys("No file argument provided", 1)
+    }
+
+    val file = new File(args(0))
+    if (!file.exists() || !file.isFile) {
+      exitSys("File does not exist", 2)
```

Comment:
FYI (not sure if this is an issue, but) when I run the panel with a nonexistent file, I don't see "File does not exist" in the log. This is what I see:

```
> run wrong_filename.txt
[info] Running hmda.panel.PanelCsvLoader /Users/lortone/dev/hmda-project/hmda-platform-2/panel/src/main/resources/inst_datajjj
[info] [DEBUG] [06/22/2017 15:24:43.446] [main] [EventStream] StandardOutLogger started
[info] [DEBUG] [06/22/2017 15:24:43.666] [main] [EventStream(akka://hmda)] logger log1-Logging$DefaultLogger started
[info] [DEBUG] [06/22/2017 15:24:43.667] [main] [EventStream(akka://hmda)] Default Loggers started
java.lang.RuntimeException: Nonzero exit code returned from runner: 2
	at scala.sys.package$.error(package.scala:27)
[trace] Stack trace suppressed: run last panel/compile:run for the full output.
[error] (panel/compile:run) Nonzero exit code returned from runner: 2
[error] Total time: 2 s, completed Jun 22, 2017 3:24:44 PM
```